dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
are you referring to the plugin versions?  there are no dependencies (besides test dependencies) in the referenced parent pom.  Should I go through the dropwizard-core pom?  or dropwizard-bom which has every version of every dependency that any part of dropwizard uses.  (i used dropwizard-bom to update any dependency versions that were already in this parent pom - i did not look for new ones...)